Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM govukpay/alpine:latest-master
+# govukpay/alpine:latest-master
+FROM govukpay/alpine@sha256:1200b7cfe9636e435f100b0f19e8bcdaccea4d5c1a073776ebe95f869705d57c
```

Comment:
Looks like we don't need the pay base image for this - raised a separate PR which switches to pulling from Alpine directly: https://github.com/alphagov/pay-notifications/pull/24